src/models/methods/bookOffers.ts

```diff
@@ -1,24 +1,24 @@
-import { Amount, LedgerIndex } from "../common";
+import { Amount, Currency, LedgerIndex } from "../common";
 import { Offer } from "../ledger";
 
 import { BaseRequest, BaseResponse } from "./baseMethod";
 
-interface TakerAmount {
-  currency: string;
-  issuer?: string;
-}
+// interface TakerAmount {
+//   currency: string;
+//   issuer?: string;
+// }
 
 export interface BookOffersRequest extends BaseRequest {
   command: "book_offers";
   ledger_hash?: string;
   ledger_index?: LedgerIndex;
   limit?: number;
   taker?: string;
-  taker_gets: TakerAmount;
-  taker_pays: TakerAmount;
+  taker_gets: Currency;
+  taker_pays: Currency;
```

Comment:
This is incorrect - XRP is given without an issuer, as just `{currency: XRP}`. So the `TakerAmount` object is needed.

Reply:
I think even this should be `Amount`. The type matches with what book_offers needs.